cypress/e2e/ui/Alert-delete-fail.cy.js

```diff
@@ -0,0 +1,52 @@
+import '@testing-library/cypress/add-commands';
+
+import { getUKFormattedDate } from '../../../src/utils/helpers';
+
+describe('ALERT DELETE EXPENSE - FAIL', () => {
+  before(() => {
+    cy.intercept('GET', '**/items*', { fixture: 'expense-array' }).as(
+      'getExpenseArray'
+    );
+    cy.visit('/');
+    cy.wait('@getExpenseArray');
+  });
+
+  beforeEach(() => {
+    cy.intercept('DELETE', '**/items/**', {
+      statusCode: 500,
+    }).as('deleteExpenseFail');
+  });
+
+  it('selects test month', () => {
+    const TEST_MONTH = 'February 2022';
```

Comment:
How come you bind the date to a `const` instead of just using it directly?

Reply:
This is probably my OCD! but nonetheless, one of the problems we had with the old tests was that we needed to go through the code and look for values to be changed manually. I was always forgetting one or two and the tests were failing. My OCD pushed me to basically remove and directly input values. 😄 

`February 2022` should be a constant. This is a value that does not change and it is fixed for the test (also all expenses we add are in February).

I will move this constant outside the describe() block to make it global. This makes even more sense I think.